terraform/database.tf

```diff
@@ -0,0 +1,37 @@
+resource "aws_db_instance" "default" {
+  allocated_storage    = 20
+  db_name              = var.DB_NAME
+  engine               = "postgres"
+  engine_version       = "16.1"
+  instance_class       = "db.t3.micro"
+  publicly_accessible          = true
+  performance_insights_enabled = false
+  skip_final_snapshot  = true
+  db_subnet_group_name         = data.aws_db_subnet_group.public_subnet_group.name
+  vpc_security_group_ids       = [aws_security_group.rds_security_group.id]
+  username             = var.DB_USER
+  password             = var.DB_PASSWORD
+}
+
+resource "aws_security_group" "rds_security_group" {
+    name = "c10-games-tracker-sg"
+    vpc_id = data.aws_vpc.cohort_10_vpc.id
+}
+
+resource "aws_security_group_rule" "allow-all-ipv4-traffic" {
+  type              = "ingress"
+  from_port         = 0
+  to_port           = 5432
+  protocol          = "tcp"
+  cidr_blocks       = ["0.0.0.0/0"]
+  security_group_id = aws_security_group.rds_security_group.id
+}
```

Comment:
I'm not 100% sure if this is right or not, but you could always use the security group made for our original RDS? 
Name: c10-games-tracker-sg
ID: sg-0669a5a1256e2e077

Reply:
Would we not want to terraform the security group and its rules?

Reply:
good point! probably best to terraform then!